django_features/serializers.py

```diff
@@ -0,0 +1,224 @@
+                            field=field, required=False
+                        ),
+                        required=False,
+                    )
+        for field_name, field in nested_fields.items():
+            self.related_fields.add(field_name)
+            fields[field_name] = NestedMappingSerializer(
+                data=nested_field_data[field_name],
+                exclude=[*self.exclude, self.model.__name__.lower()],
+                field=field,
+                nested_fields=nested_field_fields[field_name],
+                parent_mapping=self.mapping,
+                required=False,
+            )
+        self.fields = fields
+        return fields
+
+    def create(self, validated_data: dict[str, Any]) -> models.Model:
+        relations_to_save: dict[str, Any] = {}
+        for field in self.related_fields:
+            value = validated_data.pop(field, None)
+            if value is not None:
+                relations_to_save[field] = value
+            else:
+                serializer = self.fields.get(field)
+                if serializer is not None:
+                    serializer.is_valid(raise_exception=True)
+                    relations_to_save[field] = serializer.save()
+        instance = super().create(validated_data)
+        for field, value in relations_to_save.items():
+            model_field = self.model._meta.get_field(field)
+            if model_field.many_to_many or model_field.one_to_many:
+                getattr(instance, field).set(value)
+            if model_field.one_to_one or model_field.many_to_one:
+                setattr(instance, field, value)
+        instance.save()
+        return instance
+
+    def update(
+        self, instance: models.Model, validated_data: dict[str, Any]
+    ) -> models.Model:
+        for field in self.related_fields:
+            value = validated_data.pop(field, None)
+            if value is None:
+                serializer = self.fields[field]
+                serializer.is_valid(raise_exception=True)
+                value = serializer.save()
+            model_field = self.model._meta.get_field(field)
+            if model_field.many_to_many or model_field.one_to_many:
+                getattr(instance, field).set(value)
+            if model_field.one_to_one or model_field.many_to_one:
+                setattr(instance, field, value)
+        return super().update(instance, validated_data)
+
+
+class NestedMappingSerializer(BaseMappingSerializer):
+    class Meta:
+        fields = "__all__"
+        model = None
+
+    def __init__(
+        self,
+        exclude: list,
+        field: models.Field,
+        nested_fields: list,
+        parent_mapping: dict[str, Any],
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        self.exclude = exclude
+        self.field = field
+        self.nested_fields = nested_fields
+        self.parent_mapping = parent_mapping
+        self.Meta.model = self.field.related_model
```

Comment:
As far as i see field is only used here.

```suggestion
        self.nested_fields = nested_fields
        self.parent_mapping = parent_mapping
        self.Meta.model = field.related_model
```